source/elements/oneDAL/source/programming_model/computational_modes.rst

```diff
@@ -5,11 +5,18 @@ Computational modes
 -----
 Batch
```

Comment:
Start with The algorithms of the library support or similar, then follow the style we have to decide for defs in terminology section.
Also, general question how overall would look like, incl intro, this section, defs etc